cluster-scope/base/apps/daemonsets/kepler-exporter/daemonset.yaml

```diff
@@ -0,0 +1,64 @@
+apiVersion: apps/v1
+kind: DaemonSet
```

Comment:
Can we add resource restrictions on this daemonset? this will give us an idea of what sort of usage to expect on the cluster and establish a reasonable quota for the namespace